api/src/main/java/com/okta/sdk/http/HttpRequests.java

```diff
@@ -47,7 +47,7 @@
      * @return a new HttpRequestBuilder that can be used to construct a new {@link HttpRequest} instance.
      * @throws IllegalArgumentException if the method argument is {@code null}.
```

Comment:
This comment needs to be updated.

Reply:
This doc line is still actually valid. The exception is thrown when `method` is null. It's just not a checked exception